config/api.yml

```diff
@@ -176,8 +176,8 @@
       - :name: delete
         :identifier: alert_definition_delete
   :alerts:
-    :description: Alerts
-    :identifier: alert
+    :description: Alert Statuses
```

Comment:
There a reason you decided to change the name here?

This wasn't even a change in `ManageIQ/manageiq`:

https://github.com/ManageIQ/manageiq/pull/20438/files#diff-cdcd147fb85ef58d008a4ad0cfa3ddfea649382cb062407e2342aff07362b6cfL2049

so not sure what is prompting the change here.

Reply:
Because this endpoint is not for alerts but alert statuses, wrongly named (see the class and the child endpoints below)

Reply:
Fair enough.